app/views/service_providers/_form.html.erb

```diff
@@ -136,9 +136,17 @@
                      class: 'usa-label' %>
     </div>
 
-    <%= form.input :redirect_uris,
-                   input_html: { class: 'usa-input', multiple: 'multiple' },
-                   label: false %>
+    <% values = service_provider.redirect_uris || [] %>
+    <% values << nil %>
```

Comment:
is there a controller we can move this logic to? IMO it's better/more understandable to have the templates be simple view logic (ifs, loops, etc)

Reply:
Yup - good call